google_gke_infra/README.md

```diff
@@ -148,97 +152,99 @@ module "k8s" {
 }
 ```
 
-### Upgrading a Cluster
+### Upgrade a cluster
+
+The GKE module exposes two variables to allow the separate upgrade of the Control Plane and Nodes.
 
-The module exposes two variables to allow the separate upgrade of the control plane and nodes:
+#### Update **k8s_version**
 
-#### Update `k8s_version`
+>**NOTE:** This value configures the version of the Control Plane. You must upgrade it first. 
 
-This value configures the version of the Control Plane. It needs to be upgraded first. Generally this operation takes ~15 minutes. Once that is done, re-run `terraform plan` to validate that no further changes are needed by Terraform. If the node pools are configured for auto upgrade, GCP will take care of automatically upgrading the nodes within the upcoming weeks to match the master version, and hence you don't need to manually adjust `node_version`.
+The updating operation takes about 15 minutes. Once it is done, re-run `terraform plan` to validate that Terraform does not need any further changes. If the node pools are configured for auto upgrade, GCP automatically upgrades the Nodes within the upcoming weeks to match the master version, so that you don't need to manually adjust the **node_version**.
 
-#### Update `node_version`
+#### Update **node_version**
```

Comment:
It would be good to avoid level 4 headings, but I get the point that the document is quite long, that's why I left it. Maybe there is a plan to create some smaller docs out of it later? Nonetheless, it can stay as is for now, as it is still in the incubator :)